packer/oem-build.json

```diff
@@ -23,12 +24,14 @@
       "type": "virtualbox-iso",
       "headless": "{{user `headless`}}",
       "http_directory": "http",
+      "vm_name": "{{user `vm_name`}} {{user `semester`}}",
+
+      "cpus": 2,
+      "memory": 4096,
```

Comment:
Are we confident that this jump will be fine on the majority of machines? If we still have people coming in with only 4GB of RAM, this may be problematic. Not a rejection, but worth discussing

Reply:
I've been resisting this change for a while, but I think we've reached a point where a majority of people will benefit from this change, and if we do encounter a machine so under-powered, RAM is an easy enough GUI tweak after import. It's also more the 8GB machines that scare me here. I think 4GB is going to have a bad experience regardless, but at 8, we're taking 50% of the machine.

Reply:
Works for me!